packages/@n8n/task-runner-node-js/src/start.ts

```diff
@@ -1,9 +1,10 @@
 import * as a from 'node:assert/strict';
 
-import { JsTaskRunner } from './code';
 import { authenticate } from './authenticator';
+import { JsTaskRunner } from './code';
 
-let _runner: JsTaskRunner;
+let _runner: JsTaskRunner | undefined;
```

Comment:
Should probably rename this to `runner`, no?

Reply:
Yes 👍